src/app/progress/PowerCaps.tsx

```diff
@@ -0,0 +1,13 @@
+import { powerLevelByKeyword } from 'app/search/d2-known-values';
+import './powercaps.scss';
+
+export function PowerCaps({ powercap, label }: { powercap: string; label: string }) {
```

Comment:
It'd be good to constrain the `powercap` type to the possible values, e.g.
```suggestion
export function PowerCaps({ powercap, label }: { powercap: keyof typeof powerLevelByKeyword; label: string }) {
```